client/src/pages/Home.jsx

```diff
@@ -5,47 +5,68 @@ import { Icon } from "leaflet";
 import LeafletGeocoder from "../components/LeafletGeocoder";
 import "leaflet/dist/leaflet.css";
 import "../styles/Home.css";
+import ArtDetails from "../components/ArtDetails";
 
 function Home() {
   const [position, setPosition] = useState([
     44.831271602173324, -0.5722962522737938,
   ]);
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedArt, setSelectedArt] = useState(null);
 
   const artData = useLoaderData();
 
-  const artUrl = import.meta.env.VITE_API_URL;
-
   const artIcon = (url) =>
     new Icon({
       iconUrl: url,
       iconSize: [38, 38],
     });
 
+  const artUrl = import.meta.env.VITE_API_URL;
+
   useEffect(() => {
     navigator.geolocation.getCurrentPosition((geoPosition) => {
       const { latitude, longitude } = geoPosition.coords;
       setPosition([latitude, longitude]);
     });
   }, []);
 
+  const openModal = (art) => {
```

Comment:
We tend to use the prefix "handle" + "the action" for event like those. Such as "handleOpenModal".

It's not a mistake, we can continue like that

Reply:
I also considered using the "handle" prefix but the name was rather long with it... 

I just changed it to "handleOpenModal" as you suggested, which is more logical. I made the same modification for "handleCloseModal"